interfaces/builtin/basedeclaration.go

```diff
@@ -195,9 +195,28 @@ slots:
       slot-snap-type:
         - app
         - gadget
+    allow-connection:
+      -
+        plug-attributes:
+          content: $SLOT(content)
+      -
+        plug-attributes:
+          content: $MISSING
+        slot-attributes:
+          content: $MISSING
```

Comment:
Perhaps this is meant to address the 'default' from https://github.com/snapcore/snapd/wiki/Interfaces#content?

- content (slot): reference to plug side of connection. Defaults to local slot name
- content (plug): reference to slot side of connection. Defaults to local plug name

That said, it seems that the first alternation addresses the default case just fine, unless I am missing something:
```
plug-attributes:
    content: $SLOT(content)
```